### PR TITLE
[Test] Add bn layer training

### PR DESCRIPTION
**Pull request by zhoonit**
## Commits to be reviewed in this PR

<details><summary>[Test] Add bn layer training</summary><br />

**Changes proposed in this PR:**
- Add batchnormalization layer test (for channel, width axis)

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

</details>

### CODE REVIEW

File: test/input_gen/genLayerTests.py [hunk 1]
```diff
@@ -48,8 +48,13 @@ def inspect_file(file_name):
     record_single(fc, (3, 1, 1, 10), "fc_golden_plain")
     fc = K.layers.Dense(4)
     record_single(fc, (1, 1, 1, 10), "fc_golden_single_batch")
-
-# inspect_file("fc_golden.nnlayergolden")
-
-
+    bn = K.layers.BatchNormalization()
+    record_single(bn, (2, 4, 2, 3), "bn_golden_channels_training", {"training": True})
```
Comment (kparichay) [inline test/input_gen/genLayerTests.py:52]: consider increasing batch size more (just for batch normalization) - maybe 4?

**Review by kparichay — APPROVED** [review]
LGTM other than the minor comment

File: test/input_gen/genLayerTests.py (hunk 1 above)
Comment (zhoonit) [reply test/input_gen/genLayerTests.py:52]: Will update

Comment (kparichay) [reply test/input_gen/genLayerTests.py:52]: sounds good. I will ensure to rebase #1590 after this merge and check with both batch size 2 and 4.

File: test/input_gen/genLayerTests.py [hunk 2]
```diff
@@ -48,8 +48,13 @@ def inspect_file(file_name):
...
+    ## @todo add test for inference
+    record_single(bn, (2, 4, 2, 3), "bn_golden_channels_inference", {"training": False})
+    bn = K.layers.BatchNormalization()
```
Comment (lhs8928) [inline test/input_gen/genLayerTests.py:55]: We can remove this line I guess?

**Review by lhs8928 — APPROVED** [review]
LGTM

File: test/input_gen/genLayerTests.py (hunk 2 above)
Comment (zhoonit) [reply test/input_gen/genLayerTests.py:55]: input shape is different so this has to be initialized again

**Review by jijoongmoon — APPROVED** [review]
LGTM
